broadbean/broadbean.py

```diff
@@ -1850,6 +1958,163 @@ def outputForAWGFile(self):
                                                       DCgain=1)
                     elements[pos][chan][0] = postfilter
 
+        return elements
+
+    def outputForSEQXFile(self) -> Tuple:
```

Comment:
Given that this is a well defined tuple it could be fully types as Tuple[type(trig_wait), type(nreps), and so on